node/core/dispute-coordinator/src/lib.rs

```diff
@@ -118,11 +119,16 @@ pub struct DisputeCoordinatorSubsystem {
 pub struct Config {
 	/// The data column in the store to use for dispute data.
 	pub col_data: u32,
```

Comment:
With addition of the `col_session_data` this column might be renamed to something like `col_disputes_data` just for consistency.